backend/src/test/java/org/partypets/backend/service/PartyServiceTest.java

```diff
@@ -53,4 +53,17 @@ void expectId_whenAddedParty() {
     }
 
 
+    @Test
+    void expectParty_whenGettingPartyDetails() {
+        //given
+        Party expected = new Party("abc", new Date(), "Home", "Dog-Bday");
+
+        //when
+        when(partyRepo.getById("abc")).thenReturn(expected);
+        Party actual = partyService.getDetails("abc");
+
+        //then
+        assertEquals(expected, actual);
+        verify(partyRepo).getById("abc");
```

Comment:
You should use the verify() to check if the mocked. methods have been called before executing assertEquals ().
```suggestion
        verify(partyRepo).getById("abc");
        assertEquals(expected, actual);
        
```